Tasks/DotNetCoreCLI/dotnetcore.ts

```diff
@@ -101,6 +101,7 @@ export class dotNetExe {
         const resultsDirectory = tl.getVariable('Agent.TempDirectory');
 
         if (enablePublishTestResults && enablePublishTestResults === true) {
+            tl.rmRF(`${resultsDirectory}/*.trx`);
```

Comment:
rmRF doesn't support this. We need to find the files and if they exist, pass it to rmRF function. 
Please check for doc: https://github.com/Microsoft/vsts-task-lib/blob/master/node/task.ts
https://github.com/Mardoxx/vsts-tasks/blob/8b361827f14aed92de1cd0cc05df3ab136aadbcb/docs/contribute.md